docs/app/pages/components/components-sections/date-time-picker/date-range-picker/date-range-picker.component.ts

```diff
@@ -78,7 +78,7 @@ export class ComponentsDateRangePickerComponent extends BaseDocumentationSection
         // check if the date contains a hyphen
```

Comment:
In Digital Safe we are currently getting a call stack exceeded error when using the Date Time Picker component with time view enabled. I notice that there is a incorrect `if` expression:

![image](https://user-images.githubusercontent.com/28828887/73740611-64a16800-4740-11ea-9478-69fd3c82b284.png)

The comment says emit only if they are different, whereas the code is emitting only if they are the same, which is causing an infinite loop.

Might be good if we could fix this as part of this ticket? (It's Ashley btw 😁)

Reply:
Fine by me, if you have a repro case for us to add to the ticket for QA?

Reply:
I'll see if I can come up with one, thanks

Reply:
https://plnkr.co/edit/tKHxoVhZvKFJrrMzqdPb?p=preview

Reply:
If `date` is undefined before the popover is opened the error seems to occur

Reply:
Thank you @TaraTodd123 and @ashley-hunter for looking at this 😃